client/my-sites/customer-home/cards/tasks/site-setup-list/index.jsx

```diff
@@ -213,96 +211,54 @@ const SiteSetupList = ( {
 	return (
 		<Card className={ classnames( 'site-setup-list', { 'is-loading': isLoading } ) }>
 			{ isLoading && <Spinner /> }
-			{ useDrillLayout && (
-				<CardHeading>
-					<>
-						{ currentDrillLayoutView === 'task' && (
-							<Gridicon
-								icon="chevron-left"
-								size={ 18 }
-								className="site-setup-list__nav-back"
-								onClick={ () => setCurrentDrillLayoutView( 'nav' ) }
-							/>
-						) }
-						{ translate( 'Site setup' ) }
-					</>
-				</CardHeading>
+			{ useDrillLayout && <CardHeading>{ translate( 'Site setup' ) }</CardHeading> }
```

Comment:
Yes, I completely agree with you. I actually tested it but I tested opposite. Instead of removing this one, I removed the one inside nav which resulted in breaking design. But this one works just fine.